dev_settings.py

```diff
@@ -82,8 +82,8 @@
     'default': {
         'COUCH_HTTPS': False,
         'COUCH_SERVER_ROOT': 'localhost:5984',
-        'COUCH_USERNAME': '',
-        'COUCH_PASSWORD': '',
+        'COUCH_USERNAME': 'admin',
+        'COUCH_PASSWORD': 'commcarehq',
```

Comment:
Will this change break a local instance of HQ if the docker container is not also upgraded?

Reply:
It will. I wasn't able to find a solution for this. Every request made with these (or any) credentials to the current container will result in `error: unauthorized`.

Regarding your previous question, I just added a commit that should allow us to upgrade and downgrade the container without worrying about the local data, which should remain usable across versions without the need of any special operation.

Reply:
My mental read of this is that as long as people are using the `localsettings.example.py` template which imports `*` from `dev_settings.py` and their local docker containers are updated right away, I imagine no one will have to do anything extra like manually edit any settings files or anything

Reply:
It seems likely that devs will receive this change while the previous version of the docker container is still running. Any new Python  process (or existing Django web server process after settings are reloaded) that tries to access Couch after that point will fail with an authentication error until the new Couch container is started. The resolution to that error may not be obvious, so we may want to communicate to the team about how to resolve it.

Reply:
> their local docker containers are updated right away

I don't think this is automatic. It will require something like `./scripts/docker up -d couch`